### DISCUSSION TITLE
Is it possible to limit visible items in v-autocomplete?

### DISCUSSION THREAD

**Discussion by gentlekenny** [vuetifyjs/vuetify#18059] [Q&A]
I was just wondering is it possible to limit number of visible items in autocomplete?
I have 9000 items and of course, page crashes because it attempts to display a lot of items per inputted keyboard. 
I've read the doc but couldn't find a related property

**Comment by yuwu9145** [comment]
autocomplete has virtual scroll which only renders visible items to prevent crash: [10000 items demo](https://play.vuetifyjs.com/#eNqlkNFuwjAMRX/FyiYNpLYZYk8VIHjcN1A0SmtGtjaNEheEqv77nBQkBC9I60sTx9fnXq87sTImObYoUjEjrE2VEy4yDTA7xrkx4RguRaMpVxrtpTR0tNQUDauQEFLFA9w8E+GfCZBXtbyX+8owfSZvqHx1hVWGwCG1A1zVprEEHXhOS1hGYHEfwSmn4gA97G1TwxsnePN64DbtCIIFmMPK2vyc+J5RBxXqbzqkMHnnD/oIRl8RqDHMF7D9ZAG8vHaq3469q8EHOxCRGCzEdW6SH9doXlXnSRw0PHDUFELF19iJv2fiQGRcKmVRapaVWKmjTTSS1KaWS26TttWkaozLpl5Ok2nyIUvl6LacoKvjnW1ODi0PyUR0gyG1Pz+BunQGwmQ6IC61uMp3zjMeZksGHtHGFnWJFu2zke5kt7Hunh6ieXqf6Z4XXin96+52Xbiw5/XV5P8yh2l+1MYT+0gEo4weFGLzB+0JCcY=)

So the crash in your end should be something else

**Comment by b-bika** [comment]
Any ideas on why the performance of VAutocomplete may have gotten worse in a newer version of Vuetify?
If I increase the number of items to 100000 in the playground &  use Vuetify 3.3.17, everything works fine. But on v3.6.7 it's laggy. Couldn't find any prop to configure this.
